packages/api/test/helper/constants.ts

```diff
@@ -29,4 +40,42 @@ const appData = {
iEqlBpQAlQECtCeE0JwQKhCEHJIlQoSRCVIgRCVCIIhCVBT4ywmm9o3ymPT/pZ3DrxttZsqPY4saxshol3AHTxWsvWd6Ds5vvt9CFWWNm34RpuEhrntg8iZ/FcW8809TxamvHOuWC4sLql8alRfllwlwaJLTBiDzXS9zAOc6ie7oXZSYkTGizT729w95o21JtSjUfmaHZszHGM0QdWkAOjmHLS3eOVA1wfSa7NqcjiNwBGo6bq3Jwnz+XJOshjtQuIzU3DN8vcfr6BZev8ON41iSHDXlqFssR7WPc4uFpr3hq8aB3Ed3dZC9xxjwxht3SHukNh0zJBA3LtRKjM/pbV1z3OI1vWZmDWvaSdmg6+i0dlTLmODuBafMyD7AKmwTB4d8aoIIktZvlmd+uq0Fy74du53F8hvmMjfck+Sa53kRnsl6k4XU/0qUcS70A/VXttGX7+x+qzlvDMjD+4zXxdqfwVzbVhtP3utM1ybntm+2OJuZcU2NDHDJmJIJ+ZxG8j+H3Wy7E29RzDcVT3nd1jYgNbuYHUj2XnnaBgq4gWTBDabQ7kSOPm73XseGW+SmxgM5WgT4AbLXOZ3rLerziY0JwCRoTloyImuKcUwoETgkCcEChPASMC6AIGwhOQg4JEqFCSISoQIhCECISoUoRr2lmbI3aZ/MffJQqXMcR9NFbKufTyuLeDu838QufzZ7+Tp/j75fjf253Fox4Bdw4jdQa9R4/8mb+oA7qa4mCFVXTHa8eEa/VZTTun++1Pitzv3WGQB8jeHHxWUq1SXkhrWlxglrWjpwWruaBLZLY91nq1uc2gk/Tqo+S9169TjpngNYD4nw3Um57zmNdORgzkezP/AKPmoNrTzPDZ4wSrOvq50dAB0AhWmf259a56cmOJdm5meGw/6VrZMOxO+w/H6qDToganfTTRTbV8eOw9VaVjqdYfHnvGI1sgOaWAa8Pht9NgvbsHc40WF25a0kcjC83w/CGVsVfmeBDWVMhEueA1s5Z04a9F6qxsCOS3z9Off2eEFKEhV2ZpTUpSIALowJgXViBwSoQgEIQg4oQkUJCEIQIhKkUgQhCICi37O5m4tOb8CPQlSky5IDdd3GGjw1Kpu8zWmJ+UVj3DdV1epB1Uq5t3DVgkfw8vBUN7dGSCI9vSVx879PSzrn2fe3DSNSs1f3oEhsDh7qTeVp0zDxkKsYynMkknh681Oc++1G/J+okWTMoDid5gaKe0E7+J8+Hh+agseJzO0iIAO0LheYq1o7pkq9v6jCTvtYXN21vHXc8vRMtLvM8a6TssnXvXOOpVjhd61sFzgIUyIq7uajqeI0arf4Nf7Xx9HL1im8OAI4rxihdi4uczflY1rAeZLpJ9l6zZVsuUHYgDzC2x9Ofy/axTSlKaVoyCRKlCBQE9ia1PagckSykQCEIQckIKRQkFCEIBIlQpCIT203O2HmpDLUfvGUQjMYSYCou1F+GVqDZ0zhh/ua79FrQ0DYLyb/EG9IuKAHGo5x/tiFh5r+PHR/Hn59bgFQLu2a+QRPkCn2FxmY0zwSVeRXJK7blkr7C4JDR9PZUV1aubx9VvLimIJ4rO3jMxj8leaUuWSqtfMSVFfQdxWnq2yiPtFb5HxZ/9lK5vtloDaJhsi4hrRLnENA6kwE+Z8Fl2Lw2GB5/feSPBug+hXoTndyR0VPbWjaeVjdmNDR1galWVN+kLqzORw7vat7G4ztPNuhUlZZ9w+mXOYdQJA58wfKVcYdi9OqBrlceB/BWU4sQnBInBSgoTgkCEDkICVAISoQcEIStYTsFCTUQpNO2P73opDWAbBBDbQceEeKkMt2jfXxXZR61SD1QdpA0SrhRBJkqQgQheL/4h0yLik47S8T1Xslas1olzgB1WA7e4d+00S+g0vdTeHw0Eu1EHQa7Zll5Z2N/Bea9onZ697gbKvy8OWAwmuWaGQRuDoR4ha+0uJA1XE9Gz9u1y3RVT7cclcPcCo9VoUxRR1aCb+y6K0bQJKWsyFIo32ys+zlgDUdVcO7SEg/znRvoJPonNti4hrRJJAA5krRXFoKNFtJup+Z7ubjv+A8lp4s913+mXm38c8/dVGfUnmU9tcDdRKz4UKtXPBdXXDxJxO80MdPqEzD2yyeXFQntJY49J9NVKwqtBg7GCFKeemksMSezR/eHPiFeW9ZrxLTKz1NgOq6sY5plhhJVbGjCcFV22JDZ4g81ZscCJBkc1ZU5KlQgRCEIEo0c2p2+qlNaAICUCE5QkgSoQga4qBVMuUyoYBUNokqKtE5jYCbWqhoJcYASvcGiSYA4rO4vfF4hvyj38Ut4Zz2oeIXhqOLjt+6OQXO1uCyoCDAdAPgVFknYKZ+yGGkjks5/bX19LbFMOo1Wj4jA7lUgBw8HDVZi5w59A6SWcHcvFbO1thAB2hdv2FnKfFTrxzU9mPNrF/wAYRl0lNaSqypWh7gNg5wHqVItnyVxe3oc/a1two1eJVhbN0XS3tWhr6ztAwENmNXcDHGPr4K2M3V4z3qYnaZhNSnSdmeCXbBoHyzu4zx4LviVy14JYZG/X0VEK+u8+K7tpujOdOX5rrzPjORxat1rtQLifRQm0ySrN4Dt9DwPA+PJNY3KdRH3upU+nNlHQjmCoOHyMvSQrJ7wCFCayHvH80+uqJjQWynNcQNpUKxGis20+6rIRSGu2T6b6lMy06cuHooxIa4knZd7WvmbLuZEpEWLi1xFj9Hd13I7HwKmrPuocQnW9++mYfq33HgVKLF6hQP8AN6XM+iRSheIQhQBIUIQcbj5VGo/MEIUVMGM/7R8QszU+UoQq6+2mDrbdWb9vMfRKhRCri2+UeAXcoQtGTxu5+d39bvqVOw/dCF572J9NHbfKnYx/sUvF/wD7FCFr4vuub+R9RQW3zjxH1V1c/KkQt3Mqaic/5G/3fVCFMRpGO4TH/wC4fAIQiIv8P2VvT28kqFZChvPmKlUPkH3xQhRBYUuPkot9shClFQkIQiH/2Q==",
```

Comment:
What is this? You can use a smaller link like: https://i.picsum.photos/id/496/400/400.jpg?hmac=8or0mw_rugRSBRDFhLAj770EHslVmfHZEu0a_NKkxoY